server/test/lib/scene/scene.checkTrigger.test.js

```diff
@@ -482,6 +490,80 @@ describe('scene.checkTrigger', () => {
     });
   });
 
+  it('should execute scene with message received trigger', async () => {
```

Comment:
Can you create all this in the "triggers" folder (cf message line 13 of this file :D )